server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMessagesUpdateProcessor.java

```diff
@@ -139,7 +139,7 @@ private void update(MessageId messageId, UpdateMessagePatch updateMessagePatch,
         } catch (InvalidOutboxMoveException e) {
             ValidationResult invalidPropertyMailboxIds = ValidationResult.builder()
                 .property(MessageProperties.MessageProperty.mailboxIds.asFieldName())
-                .message("only drafts can be moved to Outbox")
+                .message(e.getMessage())
```

Comment:
I prefer previous solution

Reply:
InvalidOutboxMoveException fits the scope of the two assertions, I don't want to introduce pointless overspecialised exception that don't add value.

Reply:
Ok, don't mind